System.Compression.Primitives/System.IO.Compression.Primitives.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
You are missing an import of the common.props.

See other csproj: https://github.com/dotnet/corefxlab/blob/master/src/System.Text.Primitives/System.Text.Primitives.csproj#L2